common/src/transaction/ethereum/error.rs

```diff
@@ -52,6 +52,8 @@ pub enum EthError {
     ClientError(reqwest::Error),
     #[error("Cannot set http agent")]
     HttpAgentError,
+    #[error("DoBroadcastTx Error: {0}")]
+    DoBroadcastTxError(String),
```

Comment:
does this needs another variant?

Reply:
nope, i'll replacce with `BroadcastTxFail` 
and remove this one.

Reply:
will change do_broadcast... to other name and hide it.